### PR TITLE
HACDOCS-436: Creating README content for Python code sample.

### PR DESCRIPTION
**Pull request by jseseCCS**
[HACDOCS-436](https://issues.redhat.com/browse/HACDOCS-436) creating README content for Python code sample.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+**Note:** The Python code sample uses the **8081** HTTP port.
+
+Before you begin creating an application with this `devfile` code sample, it's helpful to understand the relationship between the `devfile` and `Dockerfile` and how they contribute to your build. You can find these files at the following URLs:
+
+* [Python `devfile.yaml`](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml)
+* [Python `Dockerfile`](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/docker/Dockerfile)
+
+1. The `devfile.yaml` file has an [`image-build` component](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml#L21-L27) that points to your `Dockerfile`.
+2. The [`docker/Dockerfile`](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/docker/Dockerfile) contains the instructions you need to build the code sample as a container image.
+3. The `devfile.yaml` [`kubernetes-deploy` component](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml#L28-L41) points to a `deploy.yaml` file that contains instructions for deploying the built container image.
```
Comment (maysunfaisal) [inline README.md:12]:
```suggestion
3. The `devfile.yaml` [`kubernetes-deploy` component](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml#L31-L44) points to a `deploy.yaml` file that contains instructions for deploying the built container image.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,19 @@
...
+4. The `devfile.yaml` [`deploy` command](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml#L43-L56) completes the [outerloop](https://devfile.io/docs/2.2.0/innerloop-vs-outerloop) deployment phase by pointing to the `image-build` and `kubernetes-deploy` components to create your application.
```
Comment (maysunfaisal) [inline README.md:13]:
```suggestion
4. The `devfile.yaml` [`deploy` command](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml#L46-L59) completes the [outerloop](https://devfile.io/docs/2.2.0/innerloop-vs-outerloop) deployment phase by pointing to the `image-build` and `kubernetes-deploy` components to create your application.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,19 @@
...
+# Creating an application with a Python code sample
+
+**Note:** The Python code sample uses the **8081** HTTP port.
+
+Before you begin creating an application with this `devfile` code sample, it's helpful to understand the relationship between the `devfile` and `Dockerfile` and how they contribute to your build. You can find these files at the following URLs:
+
+* [Python `devfile.yaml`](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml)
+* [Python `Dockerfile`](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/docker/Dockerfile)
+
+1. The `devfile.yaml` file has an [`image-build` component](https://github.com/devfile-samples/devfile-sample-python-basic/blob/main/devfile.yaml#L21-L27) that points to your `Dockerfile`.
```
[the remark this answers is not in the document]
Comment (jseseCCS) [reply README.md]: already done :1st_place_medal:

File: README.md (hunk 1 above)
Comment (jseseCCS) [reply README.md]: already done

File: README.md (hunk 2 above)
Comment (jseseCCS) [reply README.md]: done!

**Review by maysunfaisal — APPROVED** [review]

**Pull request merged**